src/operator/src/expr_factory.rs

```diff
@@ -181,7 +181,7 @@ pub(crate) async fn create_external_expr(
     Ok(expr)
 }
 
-/// Convert `CreateTable` statement to `CreateExpr` gRPC request.
+/// Convert `CreateTable` statement to `CreateTableExpr` gRPC request.
```

Comment:
Good catch! 👍

Reply:
Sugg: also add `[]` to those types so the language server can process them correctly